app/src/main/java/id/rizmaulana/covid19/data/model/indonesia/IndonesiaDaily.kt

```diff
@@ -0,0 +1,33 @@
+package id.rizmaulana.covid19.data.model.indonesia
+
+import com.google.gson.annotations.Expose
+import com.google.gson.annotations.SerializedName
+
+
+/**
+ * rizmaulana 22/03/20.
+ */
+data class IndonesiaDaily(
+    @Expose @SerializedName("fid")
+    val fid: Int = 0,
+    @Expose @SerializedName("harike")
+    val harike: Int = 0,
+    @Expose @SerializedName("jumlahKasusBaruperHari")
+    val jumlahKasusBaruperHari: Int = 0,
+    @Expose @SerializedName("jumlahKasusKumulatif")
+    val jumlahKasusKumulatif: Int = 0,
+    @Expose @SerializedName("jumlahPasienMeninggal")
+    val jumlahPasienMeninggal: Int = 0,
+    @Expose @SerializedName("jumlahPasienSembuh")
+    val jumlahPasienSembuh: Int = 0,
+    @Expose @SerializedName("jumlahpasiendalamperawatan")
+    val jumlahpasiendalamperawatan: Int = 0,
+    @Expose @SerializedName("persentasePasienMeninggal")
+    val persentasePasienMeninggal: Float = 0f,
+    @Expose @SerializedName("persentasePasienSembuh")
+    val persentasePasienSembuh: Float = 0f,
+    @Expose @SerializedName("persentasePasiendalamPerawatan")
+    val persentasePasiendalamPerawatan: Float = 0f,
+    @Expose @SerializedName("tanggal")
+    val tanggal: Long = 0
```

Comment:
I think the variable name better in English, could you please change it @rizmaulana? maybe I can help you to change it? how about it?

Reply:
Ah my bad, I use json data generator, so it just follow the json key, will fix it on next PR, thanks mas @pandubaraja